.github/workflows/build.yml

```diff
@@ -9,13 +9,13 @@ on:
     branches:
       - '**'
 jobs:
-  build_x86_64:
-    name: Build x86_64
+  build:
+    name: Build node ${{ matrix.node }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-13, ubuntu-22.04, ubuntu-24.04, windows-2022]
+        os: [macos-13, ubuntu-22.04, ubuntu-24.04, ubuntu-24.04-arm, windows-2022]
```

Comment:
You might wanna throw in macos-14 or 15 too to prebuild for macos on arm, and also ubuntu-22.04-arm, maybe even yolo in windows-11-arm but that last one is hella optional

Reply:
Done as a separate commit. Let's see how it goes.

Is there a reason to build on both ubuntu-22.04 and ubuntu-24.04? Aren't they compatible?

Reply:
windows-11-arm requires a matching config.h. I'm leaving it out.

Reply:
> might have different gcc versions

Ok, but for that matter ubuntu-22.04 and ubuntu-22.04-arm are equivalent. Testing one of them should be enough.